docs/src/man/reshaping_and_pivoting.md

```diff
@@ -9,51 +9,53 @@ julia> iris = CSV.read(joinpath(dirname(pathof(DataFrames)), "../test/data/iris.
 
 julia> head(iris)
 6×5 DataFrame
-│ Row │ SepalLength │ SepalWidth │ PetalLength │ PetalWidth │ Species │
-├─────┼─────────────┼────────────┼─────────────┼────────────┼─────────┤
-│ 1   │ 5.1         │ 3.5        │ 1.4         │ 0.2        │ setosa  │
-│ 2   │ 4.9         │ 3.0        │ 1.4         │ 0.2        │ setosa  │
-│ 3   │ 4.7         │ 3.2        │ 1.3         │ 0.2        │ setosa  │
-│ 4   │ 4.6         │ 3.1        │ 1.5         │ 0.2        │ setosa  │
-│ 5   │ 5.0         │ 3.6        │ 1.4         │ 0.2        │ setosa  │
-│ 6   │ 5.4         │ 3.9        │ 1.7         │ 0.4        │ setosa  │
+│ Row │ SepalLength │ SepalWidth │ PetalLength │ PetalWidth │ Species       │
+│     │ Float64⍰    │ Float64⍰   │ Float64⍰    │ Float64⍰   │ Categorical…⍰ │
```

Comment:
It would be useful to mention somewhere that `T⍰` is a shorthand for `Union{T, Missing}`. Maybe in the page about missing values? Doesn't have to be in this PR.